.github/workflows/build_hub.yml

```diff
@@ -74,21 +74,6 @@ jobs:
           # This path is from the root of the repo as needed by the plugin
           files: prime-router/build/test-results/test/**/*.xml
 
-      - name: Run Integration Tests
```

Comment:
Context: integration tests require the full metadata catalog, which needs all settings and tables, which needs the API running, which needs the containers running